src/routes/resources/kendo-japanese-pronunciations-and-definitions/+page.svelte

```diff
@@ -0,0 +1,759 @@
+<svelte:head>
+	<title>
+		Kendo Japanese Pronunciations and Definitions | Kendo Club at the University of Michigan
+	</title>
+	<meta
+		name="description"
+		content="Basic Japanese pronunciation guide and a list of Kendo vocabulary with definitions."
+	/>
+</svelte:head>
+
+<h1>Kendo Japanese Pronunciations and Definitions</h1>
+<p>
+	This document is intended to define and provide a basic pronunciation key for basic Japanese used
+	during kendo practices. The many of our definitions are taken from Jim Breen’s World Wide Web
+	Japanese Online Dictionary, which can be found at
+	<a href="https://www.edrdg.org/cgi-bin/wwwjdic/">https://www.edrdg.org/cgi-bin/wwwjdic/</a>, and
```

Comment:
It's not great to have a url in the text on a link but whatever

Reply:
Thanks, forgot to write that one down. Added that to the content issue checklist